controllers/project_reconciler.go

```diff
@@ -15,6 +15,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var projectlabels = map[string]string{
+	"app.kubernetes.io/part-of": "project",
+}
+
+const (
+	PROJECTROLEBINDINGUSERNAME     = "edit"
```

Comment:
EDITCLUSTERROLENAME

Reply:
Constant should use all capital letters and use underscore _ to separate words.

Reply:
i will remove Constant variable with capital letters and underscore _ to separate words.